changelogs/internal/newsfragments/2193.feature

```diff
@@ -0,0 +1 @@
+TODO: Placeholder
```

Comment:
this should have been removed prior to the PR merging. Will clean it up as part of spec release stuff